app/jinja_filters.py

```diff
@@ -102,19 +100,14 @@ def format_start_end_date(start_date, end_date=None):
 
 @blueprint.app_template_filter()
 def format_str_as_date_range(value):
-    from_date = format_str_as_date(value['from'])
-    to_date = format_str_as_date(value['to'])
+    from_date = format_date(value['from'])
+    to_date = format_date(value['to'])
```

Comment:
If we replace the only call to this method in `templates/partials/summary/date.html` with `format_start_end_date(answer.value.from, answer.value.to)` we can remove this method. We should also rename `format_start_end_date` to `format_date_range`.